### PR TITLE
Update cluster level logging docs for proxy access

### PR DESCRIPTION
**Pull request by satnam6502**
Update the cluster level logging documentation to reflect the changes in PR #4959

### CODE REVIEW

File: docs/logging.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ To learn how to query, fitler etc. using Kibana you might like to look at this [
 
 You can check to see if any logs are being ingested into Elasticsearch by curling against its URL:
 ```
-$ curl http://130.211.120.118:9200/_search?size=10
+curl -k -u admin:Drt3KdRGnoQL6TQM https://130.211.152.93/api/v1beta1/proxy/services/elasticsearch-logging/_search?size=10
```
Comment (roberthbailey) [inline docs/logging.md]: Presumably this is a password specific to your cluster.

Comment (satnam6502) [inline docs/logging.md]: Yes, there is a comment after this block that asks the read to get the password from the auth file generated during cluster creation time. Does this needed to be clearer?

Comment (lavalamp) [inline docs/logging.md]: I think it'd be slightly clearer if you moved that comment up above the wall of text. Your call.

Comment (satnam6502) [inline docs/logging.md]: Thanks to both of you for this good suggestion. Done.

**Comment by lavalamp** [comment]
The user:pass@ip combo doesn't appear to be for a real cluster, so LGTM

**Pull request merged**